streams/src/test/java/org/apache/kafka/streams/kstream/internals/graph/GraphGraceSearchUtilTest.java

```diff
@@ -48,12 +48,12 @@ public void shouldFailIfThereIsNoGraceAncestor() {
         final StatefulProcessorNode<String, Long> gracelessAncestor = new StatefulProcessorNode<>(
             "stateful",
             new ProcessorParameters<>(
-                () -> new Processor<Object, Object>() {
+                () -> new Processor<String, Long>() {
```

Comment:
Since I've added the missing generic parameters to the ProcessorParameters constructor args, it detected that Object/Object was not the right parameterization here.